src/main/java/com/google/jenkins/plugins/computeengine/client/ComputeClient.java

```diff
@@ -390,6 +395,34 @@ public Instance getInstance(String projectId, String zone, String instanceId) th
         }
         return instances;
     }
+    
+    public InstanceTemplate getTemplate(String projectId, String templateName) throws IOException {
+        return compute
+                .instanceTemplates()
+                .get(projectId, templateName)
+                .execute();
+    }
+    
+    public List<InstanceTemplate> getTemplates(String projectId) throws IOException {
```

Comment:
This should also have a corresponding unit test.

Reply:
Test added..
But IMHO all that mocked client tests dont have much sense :(